packages/design-system/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -29,7 +29,10 @@ import uniqueId from 'lodash.uniqueid';
  * @return {Boolean} Is this a TextField component?
  */
 function isTextField(child) {
-  return child != null && child.type === TextField;
+  // Some applications does not support child.type
+  return (
+    child && (child.type === TextField || (child.props && child.props.label && child.props.name))
```

Comment:
Have you tried looking online? After some initial research it looks like there are issues with HOC changing the name or display name
https://stackoverflow.com/questions/28218814/how-to-access-the-class-name-of-a-react-object-in-this-props-children

Reply:
https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging

Reply:
https://stackoverflow.com/questions/43913054/reactjs-react-children-foreach-can-i-get-the-child-component-name

This one seems the most promising, there's a comment that references `babel-plugin-add-react-displayname`

Reply:
Added `babel-plugin-add-react-displayname` so we can just reference 'child.type.displayName'